.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
```

Comment:
Does this really have to run daily? Weekly, or biweekly should be enough for such a repo.

Reply:
Why not?

Reply:
Because we don't expect fast pace breakings here, as we practically have very few dependencies, and it's totally unnecessary use of resource.

In fact, as I see all versions are pinned, so then there is zero sense in running the cron.

Reply:
> In fact, as I see all versions are pinned, so then there is zero sense in running the cron.

No, the action isn't fully specified (that requires a lock file, which is an upcoming PR), and the `ci` workflow uses unpinned versions of other actions that you want to know about. So testing on CRON is a good idea. The resource use is trivial IMO, but if it is really offensive we can move it to weekly.